doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -278,6 +278,12 @@ also use the ``/usr/libexec/suricata/ebpf/xdp_filter.bpf`` (in our example TCP o
     # if the ebpf filter implements a bypass function, you can set 'bypass' to
     # yes and benefit from these feature
     bypass: yes
+    # if the ebpf filter implements ERSPAN2 bypass function,
+    # you can set 'bypass-erspan2' to yes and benefit from this feature
+    bypass-erspan2: yes
```

Comment:
Does this mean that erspan traffic is never seen by Suricata?

Reply:
No, this means a rule like `alert ssh any any -> any any (sid: 32; ssh.software; content: "OpenSSH_7.9"; bypass;)` on a flow encapsulated in erspan will be bypassed by xdp instead of locally

Reply:
If the filter program we ship supports this, why does one have to opt-in for this feature? Can it not be enabled by default? Is our `xdp_filter` just an example for others to base their custom implementations on?

Reply:
> If the filter program we ship supports this ?

It gets changed with this PR ;-)

> Can it not be enabled by default?

Yes it can, just a draft for now

> Is our xdp_filter just an example for others to base their custom implementations on?

Yes, but there is a strong link with suricata with the ebpf (hash)maps and keys used (what the key is based on the addresses and ports...)